public-env-vars.toml

```diff
@@ -0,0 +1,110 @@
+# TOML file for the `/env` route of Config endpoint
+# This file is used when the Config module is enabled.
+# It specifies a list of environment variable names (`variables`) 
+# that will be exposed by the `/env` route.
+
+
+variables = [
+    "ESPRESSO_SEQUENCER_ORCHESTRATOR_URL",
+    "ESPRESSO_SEQUENCER_CDN_ENDPOINT",
+    "ESPRESSO_SEQUENCER_LIBP2P_BIND_ADDRESS",
+    "ESPRESSO_SEQUENCER_LIBP2P_ADVERTISE_ADDRESS",
+    "ESPRESSO_STATE_RELAY_SERVER_URL",
+    "ESPRESSO_SEQUENCER_PREFUNDED_BUILDER_ACCOUNTS",
+    "ESPRESSO_SEQUENCER_L1_PROVIDER",
```

Comment:
We should probably hide this as the L1 RPC URL will often have an API key in it

Reply:
remove all these? or only the L1?

Reply:
All the L1_PROVIDER URLs

Reply:
does the ESPRESSO_PROVIDER variable needs to be removed as well?